### DISCUSSION TITLE
github mobile render issue

### DISCUSSION THREAD

**Discussion by heartacker** [community/community#4509] [Mobile]
as pic

![image](https://user-images.githubusercontent.com/1876302/124548696-a6b46900-de60-11eb-815c-0fe8988b045f.png)

**Comment by heartacker** [comment]
**CAN anybody fixes this **

**Comment by jfullstackdev** [comment]
is there a message on it when it's not dark mode ?

**Reply by heartacker** [reply]
what message?
this is render error, not a crash error, why

**Reply by jfullstackdev** [reply]
I see a similar one, extra height without anything, so it seems it's a bug

**Reply by heartacker** [reply]
> I see a similar one, extra height without anything, so it seems it's a bug 

yes
please fix this

**Comment by stevepopovich** [comment]
Hi! Is this consistently reproducible? Can you please share what version of the app your are using?

**Reply by heartacker** [reply]
I have meet this issue since 6 Jul 2021

so all the version have the issue

**Reply by heartacker** [reply]
consistently

**Reply by heartacker** [reply]
GitHub Android v1.75.0; OS SDK v27; smartisan DE106

**Reply by stevepopovich** [reply]
Thanks! I have passed this on to them for prioritization and investigation.

**Reply by alcere** [reply]
@heartacker We are struggling to reproduce this issue on our end. Can you share a link to an issue or pr where you can systematically reproduce this? Thanks!

**Reply by heartacker** [reply]
https://github.com/microsoft/vscode/issues/98618

can not upload image use mobile

another bug

**Reply by heartacker** [reply]
![6ba636f61028781ffbbd889ab26b843](https://user-images.githubusercontent.com/1876302/182406847-ceff51e3-9b73-4dea-829e-435fbe9283a3.jpg)

**Comment by heartacker** [comment]
@alcere 

https://user-images.githubusercontent.com/1876302/182474741-8298f15d-8a97-49d7-b5a8-c019f7e13b36.mp4

**Reply by alcere** [reply]
Thanks for sharing @heartacker! Apparently it is a recurrent issue for you, that's very helpful!
